Src/DependencyCollector/Shared/Implementation/RemoteDependencyConstants.cs

```diff
@@ -4,6 +4,7 @@ internal static class RemoteDependencyConstants
     {
         public const string SQL = "SQL";
         public const string HTTP = "Http";
+        public const string AI = "ApplicationInsights";
```

Comment:
What's the problem with a long descriptive value name? 
I don't like App because it doesn't make abundantly clear to the customer that this is a dependency type that says I called another app that's writing to an application insights resource. 
AIKey might be too cryptic.